src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -1727,7 +1726,7 @@ def test_sql_vnet_mgmt(self, resource_group, resource_group_location, server):
                      JMESPathCheck('resourceGroup', rg)])
 
         # test sql server vnet-rule create using subnet id
-        self.cmd('sql server vnet-rule create --name {} -g {} --server {} --subnet {}'
+        self.cmd('sql server vnet-rule create --name {} -g {} --server {} --subnet {} -i'
                  .format(vnet_rule_2, rg, server, vnet_id_1),
                  checks=[
                      JMESPathCheck('name', vnet_rule_2),
```

Comment:
add check that ignoreMissingVnetServiceEndpoint property is true (this should not require re-recording)

Reply:
I added the checks, but the test won't pass until tomorrow. A change to return the correct value for ignoreMissingVnetServiceEndpoint is in MS49 and won't be deployed in Canary until tomorrow.